travis/scripts/04-tests.sh

```diff
@@ -34,12 +34,16 @@ fi
 cd "$APP_FOLDER"
 if [ -f "mvnw" ]; then
     ./mvnw test \
-        -Dlogging.level.io.github.jhipster.sample=ERROR \
-        -Dlogging.level.io.github.jhipster.travis=ERROR
+        -Dlogging.level.org.zalando=OFF \
```

Comment:
@pascalgrimaud wouldn't this be an issue f there are any errors thrown during failing tests?